azext_iot/central/_help.py

```diff
@@ -136,6 +136,51 @@ def _load_central_devices_help():
                 az iot central app device registration-summary --app-id {appid}
         """
 
+    helps[
+        "iot central app device run-command"
+    ] = """
+            type: command
+            short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
+            long-summary: |
+                Note: payload should be nested under "request".
+                i.e. if your device expects the payload in a shape {"key": "value"}
+                payload should be {"request": {"key": "value"}}.
+                --content can be pointed at a filepath too (.../path/to/payload.json)
+            examples:
+            - name: Run command response
+              text: >
+                az iot central app device run-command
+                --app-id {appid}
+                --device-id {deviceid}
+                --interface-id {interfaceid}
+                --command-name {commandname}
+                --content {payload}
+
+            - name: Short Run command response
+              text: >
+                az iot central app device run-command
+                -n {appid}
+                -d {deviceid}
+                -i {interfaceid}
+                --cn {commandname}
+                -k {payload}
+        """
+
+    helps[
+        "iot central app device show-command-history"
+    ] = """
+            type: command
+            short-summary: Get most recent command-response request and response payload.
```

Comment:
There is currently an API limitation which is limiting the result of this operation to one item (the most recent) right?

Reply:
Correct

Reply:
I foresee this needing a top/last X # of command history

Reply:
I think its a matter of exposing it via public API, I'm seeing we have this functionality in Web UI at the moment. Might be a v-NEXT thing

![image](https://user-images.githubusercontent.com/42591144/86289739-108c9980-bba1-11ea-934f-47bb64dda22c.png)

Reply:
for now #wontfix